test/Microsoft.Azure.Devices.Edge.Test.Common/certs/EdgeCertificates.cs

```diff
@@ -20,7 +16,28 @@ public class EdgeCertificates
                 new X509Certificate2(X509Certificate.CreateFromCertFile(this.TrustedCertificatesPath))
             };
 
+        private string[] GetEdgeCertFileLocation(string deviceId)
```

Comment:
See my comment in `Certificates.GetFileLocation()`...

Reply:
The hard-coded value have been centralized to FixedPaths.cs